src/site/includes/debug.drupal.liquid

```diff
@@ -1,4 +1,4 @@
-{% if buildtype == 'localhost' %}
+{% if buildtype != 'vagovprod' %}
```

Comment:
Per our discussion, the console will contain the Metalsmith data structure for all envs outside of prod.